index.html

```diff
@@ -196,6 +196,40 @@ <h2 class="project-title">Skills</h2>
                 </div>
             </section>
 
+            <!-- Form Section -->
+            <footer id="form-session">
+                <div class="container">
+                  <h1 class="supporting-text">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                  </h1>
+                  <form action="https://formspree.io/f/mjvznanj" method="post" id="form">
+                        <input type="text" id="name" placeholder="Full name" required maxlength="30" name="name">
+                        <input type="email" id="email" placeholder="Email address" required name="email">
+                        <textarea name="" id="message" cols="30" rows="10" placeholder="Enter Text Here" required maxlength="500" name="message"></textarea>
+                        <button>Get in touch</button>
+                  </form>
```

Comment:
- [ ] Form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html) ✔.
- [ ] All input fields are wrapped in a single `form` element ✔.
- [ ] All input fields have the correct associated placeholder text and type attribute ✔.